src/Evolution/DiscontinuousGalerkin/Limiters/MinmodType.hpp

```diff
@@ -17,8 +17,7 @@ namespace Limiters {
 /// \see Limiters::Minmod for a description and reference.
 enum class MinmodType { LambdaPi1, LambdaPiN, Muscl };
 
-std::ostream& operator<<(std::ostream& os,
-                         const Limiters::MinmodType& minmod_type);
+std::ostream& operator<<(std::ostream& os, Limiters::MinmodType minmod_type);
```

Comment:
Should/could be `noexcept`. I'll let you decide whether you want to change it.

Reply:
I have some test failures to address, so I'll fix this too :)